### PR TITLE
Release v2.1.0

### PR DESCRIPTION
**Pull request by lauramaxwell**
## Overview

release tag here: https://github.com/Gilead-BioStats/gsm/releases/edit/untagged-e28be25b5e3bd8ba3b9a

## Connected Issues

- Closes #XXX

### CODE REVIEW

File: inst/workflow/1_mappings/ENROLL.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      required: true
+      type: character
+    invid:
+      required: true
+      type: character
+    country:
+      required: true
+      type: character
+    subjectid:
+      required: yes
```
Comment (zdz2101) [inline inst/workflow/1_mappings/ENROLL.yaml:18]: @lauramaxwell super minor but did we want to change this stiill?

Comment (lauramaxwell) [reply inst/workflow/1_mappings/ENROLL.yaml:18]: oh. yeah- i'd like to standardize those. i'll replace these with true in a new branch

**Review by jonthegeek — APPROVED** [review]
Looks great!

File: vignettes/gsmExtensions.Rmd [hunk 2]
```diff
@@ -0,0 +1,412 @@
...
+- [12 Standard gsm KRIs](https://github.com/Gilead-BioStats/gsm/tree/dev/inst/workflow/metrics) (e.g. [Adverse Event KRI Metric](https://github.com/Gilead-BioStats/gsm/tree/dev/inst/workflow/metrics/kri0001.yaml))
+- [Site-](https://github.com/Gilead-BioStats/gsm/blob/dev/inst/workflow/reports/report_kri_site.yaml) and [Country-level KRI Report](https://github.com/Gilead-BioStats/gsm/blob/dev/inst/workflow/reports/report_kri_country.yaml)
+
+## `meta` Specification
+
+The `meta` section of a workflow YAML provides key metadata describing the module. It must include the following fields: 
+
+- `Name`: Name of the reporting output.
+- `Type`: "Report", "Metric" or "App"
+- `ID`: The unique ID for the module. This should be the same as the workflow file name, without extension.
```
Comment (zdz2101) [inline vignettes/gsmExtensions.Rmd:46]: should we add a "required" tag of some sort for `Type` and `ID` ? As it does trigger those cli_abort()'s in ` MakeWorkflowListI()`

File: vignettes/gsmExtensions.Rmd [hunk 3]
```diff
@@ -0,0 +1,412 @@
...
+
+
+
+
+### `meta` example
+
+A simple `meta` section for a report might look like this:
+```
+  File: report_example.yaml
+  ID: example_report
```
Comment (zdz2101) [inline vignettes/gsmExtensions.Rmd:82]:
```suggestion
  ID: report_example
```

**Review by zdz2101 — COMMENTED** [review]
Couple last small comments related to the decisions made in `MakeWorkflowlist()`

File: vignettes/gsmExtensions.Rmd (hunk 2 above)
Comment (lauramaxwell) [reply vignettes/gsmExtensions.Rmd:46]: The istructions above say that the meta section "must include the following fields", which i think is sufficient

File: inst/workflow/1_mappings/ENROLL.yaml [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  Raw_ENROLL:
+    studyid:
+      required: true
+      type: character
+    invid:
+      required: true
+      type: character
+    country:
+      required: true
+      type: character
```
Comment (samussiah) [inline inst/workflow/1_mappings/ENROLL.yaml:16]:
```suggestion
      type: character
    subjid:
      required: true
      type: character
```

Comment (samussiah) [reply inst/workflow/1_mappings/ENROLL.yaml:16]: To adhere to the "common subject ID" principle.

**Review by samussiah — CHANGES_REQUESTED** [review]
minor update to the `enroll` workflow but this is looking real nice!

**Review by samussiah — APPROVED** [review]

**Pull request merged**